### DISCUSSION TITLE
Getting Tiled up and running

### DISCUSSION THREAD

**Discussion by depsypher** [excaliburjs/Excalibur#2735] [Q&A] [answered]
Trying to get a simple tile map working in a webpack project and keep getting this:
```
Cannot read properties of undefined (reading 'length')
TypeError: Cannot read properties of undefined (reading 'length')
    at B._addTiledText (http://localhost:8080/vendors-node_modules_excaliburjs_plugin-tiled_dist_excalibur-tiled_min_js-node_modules_webpac-9180a4.js:29820:77793)
    at B.addTiledMapToScene (http://localhost:8080/vendors-node_modules_excaliburjs_plugin-tiled_dist_excalibur-tiled_min_js-node_modules_webpac-9180a4.js:29820:79860)
    at http://localhost:8080/main.js:72:59
```

Is that enough of a clue for someone here to see what the issue is? Alternatively, is there any recent bare-bones example project available I can crib from? The one pointed to in the docs is pretty out of date so I've been trying to lift out what I need from it into a new project, but so far no go.

**Comment by eonarheim** [comment, marked as answer]
Hi @depsypher thanks for posting! 

I was able to reproduce this exact error when there is an Object layer with no objects! Which definitely seems like a bug (I'll make an issue soon). The workaround is to make sure all the object layers have objects.

Here is a brand new webpack sample! https://github.com/excaliburjs/sample-tiled-webpack

Additionally I do have a similar parcel example I did for a stream https://github.com/eonarheim/zelda-like

**Reply by depsypher** [reply]
Wow, thanks this is perfect. Fwiw I had no object layer at all, not even an empty one.

Your example is really helpful, now I see having a camera and player object would be the next two things I'd need anyway. I was trying to keep it really simple while climbing the learning curve and just get a tilemap rendered onscreen. Thanks for the excellent and quick answer!

**Reply by eonarheim** [reply]
> Fwiw I had no object layer at all, not even an empty one.

Oh that's good info, I'll include that in my ticket. I should be able to sneak that fix into the next version of the plugin

**Reply by eonarheim** [reply]
https://github.com/excaliburjs/excalibur-tiled/issues/453
